packages/storage/src/common/urls.ts

```diff
@@ -6,9 +6,7 @@ import { GatewayUrls } from "../types";
 export const DEFAULT_GATEWAY_URLS: GatewayUrls = {
   // Note: Gateway URLs should have trailing slashes (we clean this on user input)
   "ipfs://": [
-    "https://ipfs-3.thirdwebcdn.com/ipfs/",
-    "https://ipfs-4.thirdwebcdn.com/ipfs/",
-    "https://ipfs-5.thirdwebcdn.com/ipfs/",
+    "https://ipfs-staging.thirdwebcdn.com/ipfs/",
```

Comment:
how come we're using staging here feels off?

Reply:
this is actually blocking, we cannot hard-code staging here, we can however now change this to ipfs-public?